packages/angular_devkit/build_angular/src/browser/schema.json

```diff
@@ -41,6 +41,17 @@
         "$ref": "#/definitions/extraEntryPoint"
       }
     },
+    "inlineStyleLanguage": {
```

Comment:
Drive by comment.

I think this option needs to be added in karma and server builder as well.